src/main/java/com/icthh/xm/tmf/ms/activation/utils/TenantUtils.java

```diff
@@ -25,6 +28,17 @@ public String getTenantKey() {
         return TenantContextUtils.getRequiredTenantKeyValue(tenantContextHolder);
     }
 
+    /**
+     * Run code in context with specified tenant. Keep in mind, that after task is executed tenant context
+     * will be destroyed.
+     * NOTICE: If run before application start up: consider to run in new thread (or setting
```

Comment:
as we skipped the idea bout new thread then need to correct this readme

Reply:
@amedvedchuk done